testapp/build.gradle

```diff
@@ -104,6 +104,7 @@ dependencies {
     implementation "androidx.core:core-ktx:$androidx_coreKtx"
     implementation "androidx.lifecycle:lifecycle-extensions:$androidx_lifecycle"
     implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$androidx_lifecycle"
+    implementation "androidx.activity:activity:$androidx_activity"
```

Comment:
Why do we need this dependency and why we choose `1.2.0-alpha04`?

Reply:
I am using `registerForActivityResult` instead of `startActivityForResult` by using this dependency. See [this](https://developer.android.com/training/basics/intents/result).
I'll update it to current stable version.

Reply:
I think we should avoid using alpha or beta version because there could be bugs from google. However, if it is required then it's okay.